connection_tracker/README.md

```diff
@@ -11,10 +11,10 @@ The project provides a real-time list of connections that have been completed al
 
 ## Demo Steps
 ### Native eBPF program generation
-Note: These steps are already performed as part of the build process for the project and are repeated here for clarity.
+``SKIP:`` These steps are already performed as part of the build process for the project and are repeated here for clarity. Move to the next section to begin.
```

Comment:
the text has specific steps to follow and says not to follow them, so what's the point in using imperative language?

Reply:
Thanks @dthaler for the comment. Agree, it is confusing. I have removed that section, to keep it simpler.